src/components/user-settings.tsx

```diff
@@ -235,58 +235,96 @@ export default function UserSettings() {
 		}
 	}, [ allSnapshots, deleteAllSnapshots, __ ] );
 
+	const NonAuthenticatedAccountTab = () => (
+		<>
+			<div className="justify-between items-center w-full h-auto flex">
+				<WordPressLogo />
+				<Tooltip disabled={ ! isOffline } icon={ offlineIcon } text={ offlineMessage }>
+					<Button
+						aria-description={ isOffline ? offlineMessage : '' }
+						aria-disabled={ isOffline }
+						variant="primary"
+						onClick={ () => {
+							if ( isOffline ) {
+								return;
+							}
+							authenticate();
+						} }
+					>
+						{ __( 'Log in' ) }
+					</Button>
+				</Tooltip>
+			</div>
+			<div className="border-t border-[#F0F0F0] w-full"></div>
+		</>
+	);
+
+	const AccountTab = () => <UserInfo onLogout={ logout } user={ user } />;
+
+	const PreferencesTab = () => <LanguagePicker />;
+
+	const UsageTab = () => (
+		<>
+			<SnapshotInfo
+				isDeleting={ loadingDeletingAllSnapshots }
+				isDisabled={
+					activeSnapshotCount === 0 ||
+					loadingDeletingAllSnapshots ||
+					isLoadingAllSnapshots ||
+					isLoadingSnapshotUsage ||
+					allSnapshots?.length === 0 ||
+					isOffline
+				}
+				siteCount={ activeSnapshotCount }
+				siteLimit={ snapshotQuota }
+				onRemoveSnapshots={ onRemoveSnapshots }
+			/>
+			<PromptInfo />
+		</>
+	);
+
+	const tabs = [
+		{
+			name: 'account',
+			title: __( 'Account' ),
```

Comment:
Although the designs in #63 specify `Accounts` (plural) I opted for `Account` for now as there isn't a way to connect a Pressable account just yet, only a single WordPress.com account.